libcontainer/console_linux.go

```diff
@@ -123,8 +123,8 @@ func unlockpt(f *os.File) error {
 
 // ptsname retrieves the name of the first available pts for the given master.
 func ptsname(f *os.File) (string, error) {
-	var n int32
-	if err := ioctl(f.Fd(), unix.TIOCGPTN, uintptr(unsafe.Pointer(&n))); err != nil {
+	n, err := unix.IoctlGetInt(int(f.Fd()), unix.TIOCGPTN)
```

Comment:
Thanks ! Do you also want to make the same changes to https://github.com/containerd/console/blob/master/tc_linux.go ?